lib/supabase/accounts/createAccount.ts

```diff
@@ -1,26 +1,19 @@
 import supabase from "@/lib/supabase/serverClient";
-import type { AccountType } from "@/types/AccountType";
 import type { Tables } from "@/types/database.types";
 
-/** Account row from database with account_type field */
-export type AccountRow = Tables<"accounts"> & {
-  account_type?: AccountType | null;
-};
+/** Account row from database */
+export type AccountRow = Tables<"accounts">;
 
 /**
  * Create a new account in the database
  * @param name Name of the account to create
- * @param accountType Type of account: 'customer', 'artist', 'workspace', 'organization', 'campaign'
  * @returns Created account data or null if creation failed
  */
-export async function createAccount(
-  name: string,
-  accountType: AccountType = "artist"
-): Promise<AccountRow | null> {
+export async function createAccount(name: string): Promise<AccountRow | null> {
   try {
     const { data, error } = await supabase
       .from("accounts")
-      .insert({ name, account_type: accountType })
```

Comment:
I very much appreciate you keeping our core `accounts` primitive as simple as possible. 🙏